packages/main/src/themes/ColorPalette.css

```diff
@@ -2,11 +2,41 @@
 	display: inline-block;
 }
 
+:host(:not([hidden])[popup-mode]) {
+	width: 100%;
+}
+
+
 .ui5-cp-root {
 	display: flex;
 	flex-direction: column;
 }
 
+.ui5-cp-root.ui5-cp-root-phone,
+.ui5-cp-separator,
+.ui5-cp-root.ui5-cp-root-phone .ui5-cp-more-colors-wrapper,
+.ui5-cp-root.ui5-cp-root-phone .ui5-cp-default-color-button-wrapper,
+.ui5-cp-recent-colors-wrapper {
+	width: 100%;
+}
+
+.ui5-cp-root.ui5-cp-root-phone .ui5-cp-item-container {
+	width: 18.5rem;
+	max-width: 19.5rem;
+	max-height: 13rem;
+}
+
+.ui5-cp-recent-colors-wrapper {
```

Comment:
why flex? the wrapper contains separator and a container, both block elements which do not grow or shrink

Reply:
Because of the alignment, flex-direction.